Codeinwp/neve
# Small fixes

### Summary
Fixes for:
- image alignment sidebar issue;
- remove hamburger menu on mobile if no menu is selected;
- unreasonably long words everywhere;


### Will affect visual aspect of the product

NO

### Test instructions


- Check the `Markup: Image Alignment` post from theme unit test content with a sidebar. Should look ok.
- Remove the primary menu and check if the hamburger is still shown.
- Insert extremely long words anywhere. They should be broken into separate lines. I'd suggest [`Lopadotemachoselachogaleokranioleipsanodrimhypotrimmatosilphioparaomelitokatakechymenokichlepikossyphophattoperisteralektryonoptekephalliokigklopeleiolagoiosiraiobaphetraganopterygon`](https://en.wikipedia.org/wiki/Lopadotemachoselachogaleokranioleipsanodrimhypotrimmatosilphioparaomelitokatakechymenokichlepikossyphophattoperisteralektryonoptekephalliokigklopeleiolagoiosiraiobaphetraganopterygon).


Closes #121, #58, #61.
